pkg/lang/ir/v0/compile.go

```diff
@@ -166,6 +170,11 @@ func (g generalGraph) Labels() (map[string]string, error) {
 		return labels, err
 	}
 	labels[types.RuntimeGraphCode] = code
+	code, err = g.Dump()
+	if err != nil {
+		return labels, err
+	}
+	labels[types.GeneralGraphCode] = code
```

Comment:
I'm not sure about this design. Do we need to dump the whole graph? I thought we already have all the necessary information in labels.

Reply:
LGTM. We can create a new issue to track this.